seq2science/schemas/multiqc_config.yaml

```diff
@@ -49,14 +49,20 @@ report_section_order:
   explanation:
     order: 1000
   dupRadar:
-    order: -895
+    order: -891
+  Sample_distance_clustering:
+    order: -892
+  Spearman_correlation_clustering:
+    order: -893
+  Pearson_correlation_clustering:
+    order: -894
```

Comment:
It's minor, but this naming is sorta confusing with the deeptools clustering. Deeptools outputs this image:

![image](https://user-images.githubusercontent.com/15323399/120114535-e01df880-c17f-11eb-9380-b98cfb5fb45d.png)

Which is kinda the same, but based on reads over all regions. 

Should we call them `deseq_spearmap_correlation_clustering` and `deeptools_spearmap_correlation_clustering`?

Also I think the ordering should be that deseq2 comes after deeptools, since the deeptools plot is based on sliding window (no peak calling) and the deseq2 differential peaks (for the genomic workflows).

Reply:
Agreed on both counts. I've did some renaming in the qc.smk and the multiqc schema. Let me know if it was too much!